action.yml

```diff
@@ -11,6 +11,6 @@ inputs:
 runs:
   using: composite
   steps:
-  - name: add deadsnakes ppa and install ${{ inputs.python-version }} ${{ inputs.debug && '(debug)' || '' }}
-    run: ${{ github.action_path }}/bin/install-python ${{ inputs.python-version }} ${{ inputs.debug && '--debug' || '' }}
+  - name: add deadsnakes ppa and install ${{ inputs.python-version }} ${{ (inputs.debug == 'true' || inputs.debug == '1') && '(debug)' || '' }}
+    run: ${{ github.action_path }}/bin/install-python ${{ inputs.python-version }} ${{ (inputs.debug == 'true' || inputs.debug == '1') && '--debug' || '' }}
```

Comment:
Let's only document and allow one spelling: `true`

Reply:
Alright, let me fix that.

(I was +- following https://github.com/Jaid/get-boolean-action-input, for the record)

Reply:
Done. Let me quickly rerun pybind/pybind11#2792, just to be sure.

Reply:
Wow, still wanted to add this, but already merged :-)
https://github.com/pybind/pybind11/runs/1698682850#step:3:76